### PR TITLE
[SYM-109] Enable customization of styling of PIN input width and height

### PR DESCRIPTION
**Pull request by milosbswe**
## What
- Enable customization of styling of PIN input width and height

Screenshot 2023-09-03 at 17 45 07

## Test Plan
- ✅  iOS QA Tests passed locally
- ✅ Unit Tests passed locally

### CODE REVIEW

**Review by djoksimo — CHANGES_REQUESTED** [review]
Poštovani @milosbswe,

Thank you for these changes! I've tested your changes locally against VGS and Basis Theory and they look visually correct 🎉 .

image

I noticed the following editor warnings in Xcode that I thought might be worth looking into. Any ideas whether the cause of this constraint violation is on our end or Basis Theory's end?

<details>
<summary>
Xcode warnings when I hardcode `vaultType = VaultType.btVaultType`
</summary>

```
2023-09-05 19:35:40.730680-0400 SampleForageSDK[58768:2270631] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x60000318d720 BasisTheoryElements.TextElementUITextField:0x144019a00.width == 342   (active)>",
    "<NSLayoutConstraint:0x60000316d900 H:|-(0)-[BasisTheoryElements.TextElementUITextField:0x144019a00]   (active, names: '|':ForageSDK.BasisTheoryTextFieldWrapper:0x13ff0f330 )>",
    "<NSLayoutConstraint:0x60000316da90 H:[BasisTheoryElements.TextElementUITextField:0x144019a00]-(0)-|   (active, names: '|':ForageSDK.BasisTheoryTextFieldWrapper:0x13ff0f330 )>",
    "<NSLayoutConstraint:0x60000316d9f0 '_UITemporaryLayoutWidth' ForageSDK.BasisTheoryTextFieldWrapper:0x13ff0f330.width == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x60000318d720 BasisTheoryElements.TextElementUITextField:0x144019a00.width == 342   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
2023-09-05 19:35:40.731444-0400 SampleForageSDK[58768:2270631] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x60000318d540 BasisTheoryElements.TextElementUITextField:0x144019a00.height == 36   (active)>",
    "<NSLayoutConstraint:0x60000316da40 V:|-(0)-[BasisTheoryElements.TextElementUITextField:0x144019a00]   (active, names: '|':ForageSDK.BasisTheoryTextFieldWrapper:0x13ff0f330 )>",
    "<NSLayoutConstraint:0x60000316dc20 V:[BasisTheoryElements.TextElementUITextField:0x144019a00]-(0)-|   (active, names: '|':ForageSDK.BasisTheoryTextFieldWrapper:0x13ff0f330 )>",
    "<NSLayoutConstraint:0x6000031682d0 '_UITemporaryLayoutHeight' ForageSDK.BasisTheoryTextFieldWrapper:0x13ff0f330.height == 0   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x60000318d540 BasisTheoryElements.TextElementUITextField:0x144019a00.height == 36   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKitCore/UIView.h> may also be helpful.
```
</details>

File: Sources/ForageSDK/Component/ForagePINTextField/ForagePINTextField.swift [hunk 1]
```diff
@@ -105,6 +105,34 @@ public class ForagePINTextField: UIView, Identifiable, ForageElement {
...
+    @IBInspectable public var inputHeight: CGFloat {
+        get { return textField.inputHeight }
+        set {
+            container.alignment = .center
+            textField.inputHeight = newValue
+        }
+    }
+    
+    /// ElementHeight for the text field
+    private var _elementHeight: CGFloat = 0
```
Comment (djoksimo) [inline Sources/ForageSDK/Component/ForagePINTextField/ForagePINTextField.swift:127]: For each of these new members' comments (both PIN + PAN comments), can we make the distinction between element height and inputHeight more explicit?

For example, something like:

```
before:
/// InputHeight for the text field
inputHeight...

after:
/// Height of the input field within the ForagePANTextField.
/// - SeeAlso: `elementHeight` to customize the height of the entire ForagePANTextField.
inputHeight...

--------------

before:
/// ElementHeight for the text field
elementHeight...

after:
/// Height of the entire ForagePANTextField element.
/// - SeeAlso: `inputHeight` to customize the height of the input field within the ForagePANTextField.
elementHeight...

or 

/// Container height of the ForagePANTextField element.
/// - SeeAlso: `inputHeight` to customize the height of the input field within the ForagePANTextField.
elementHeight...
```

**Review by djoksimo — CHANGES_REQUESTED** [review]

File: Sources/ForageSDK/Component/ForagePINTextField/ForagePINTextField.swift (hunk 1 above)
Comment (milosbswe) [reply Sources/ForageSDK/Component/ForagePINTextField/ForagePINTextField.swift]: Updated. Thanks

**Review by djoksimo — APPROVED** [review]
LGTM once rebased against origin/main!

**Pull request merged**